internal/controller/server/handlers/msg_handler.go

```diff
@@ -3,22 +3,55 @@ package handlers
 import (
 	"log"
 	"net/http"
+	"remote/internal/controller/server/adapter"
+	"remote/internal/model"
+
+	"github.com/gorilla/websocket"
 )
 
-func HandleMessage(w http.ResponseWriter, r *http.Request) {
-	c, err := Connect(w, r)
-	if err != nil {
-		log.Fatalln(err)
-	}
+func HandleTableMessages(game model.Game) http.HandlerFunc {
```

Comment:
I would propose looking for a better name for this function. `HandleTableMessages` suggest that this function is handling Table messages but in reality, it returns a handler function that can handle Table messages :D Maybe `TableMessagesHandler`? Simple change - huge difference.

But if I may be honest the package is called `handler` so maybe better name for this function would be just `TableMessages` or simply `Build`? Then calling it would look like `handlers.Build(args...)`/`handlers.TableMessages(args...)` but not `handlers.HandleTableMessages(...)`

Reply:
Changed to `TableMessages`